src/ucp/core/ucp_request.h

```diff
@@ -68,6 +69,7 @@ typedef struct ucp_send_state {
  */
 typedef struct ucp_request {
     uint16_t                      flags;   /* Request flags */
+    uct_pending_req_t             uct;     /* Pending request */
```

Comment:
I need to access this for both sends and rma, so I need it under one common struct.